loadimpactcli/test_commands.py

```diff
@@ -0,0 +1,134 @@
+# coding=utf-8
+"""
+Copyright 2016 Load Impact
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from operator import attrgetter, methodcaller
+
+import click
+
+from loadimpact3.exceptions import ConnectionError
+from .client import client
+from .util import TestRunStatus, Metric, DefaultMetricType
+
+
+@click.group()
+@click.pass_context
+def test(ctx):
+    pass
+
+
+@test.command('list', short_help='List tests.')
+@click.option('--project_id', 'project_ids', multiple=True, help='Id of the project to list tests from.')
+def list_tests(project_ids):
```

Comment:
Could we limit this to the 10-20 or so tests with the most recent test runs and then add a line of output saying how to print/list more tests (using an `--all` or `--limit` flag perhaps)? This would greatly enhance the UX for someone like me who has 100s of tests across orgs and projects 😄